workflow/rules/test_cases.smk

```diff
@@ -0,0 +1,34 @@
+rule test_non_cov2:
+    input:
+        pangolin=get_non_cov2_calls(from_caller="pangolin"),
+        kallisto=get_non_cov2_calls(from_caller="kallisto"),
+        call_plots=expand(
+            "results/test-cases/plots/strain-calls/non-cov2-{accession}.strains.{caller}.svg",
+            accession=get_non_cov2_accessions(),
+            caller=["pangolin", "kallisto"],
+        ),
+    output:
+        "results/test-cases/non-sars-cov-2.csv",
+    log:
+        "../logs/test-cases/summarize_non_cov2.log",
+    conda:
+        "../envs/python.yaml"
+    script:
+        "../scripts/summarize-non-cov2.py"
+
+
+rule report_non_cov2:
```

Comment:
I don't think the benchmarks belong into the report. Ideally, I'd just evaluate evaluate them in the CI, like the other benchmarks are evaluated as well.

Reply:
This rule was just for the report I sent in to the matter most chat. However, I would like to keep it - it will be useful when we start working on the paper. It just nicely aggregates the non-cov2 test.

Regarding the CI, I already prepare the utilization of the .csv for GitHub actions and activated it with the commit 8136e2d.

Reply:
Ok, right, makes sense to me. We can have a target rule that collects all benchmark results, and then when using that together with --report you get a benchmark-only report.